rpc/src/v1/impls/eth_signing.rs

```diff
@@ -59,6 +61,7 @@ impl<C, M> EthSigningQueueClient<C, M> where C: MiningBlockChainClient, M: Miner
 			accounts: Arc::downgrade(accounts),
 			client: Arc::downgrade(client),
 			miner: Arc::downgrade(miner),
+			pending: Mutex::new(HashMap::new()),
```

Comment:
It can leak, maybe `TransientHashMap` (the same one is used for filters) could be used here?